pkg/admission/interfaces.go

```diff
@@ -64,6 +64,7 @@ type Operation string
 const (
 	Create  Operation = "CREATE"
 	Update  Operation = "UPDATE"
+	Patch   Operation = "PATCH"
```

Comment:
> Patch and update are the same thing semantically, why do they need separate entries here?

An update request has the "new object I want use", but a patch only has "here's the diff of what I want".  Patch doesn't produce the "new object I want to use", until after admission is over.